packages/bpk-tokens/src/web/base/ratings.json

```
@@ -0,0 +1,14 @@
+{
+  "imports": [
+    "./aliases.json"
+  ],
+  "global": {
+    "category": "ratings"
+  },
+  "props": {
+    "RATINGS_BASE_BORDER_RADIUS": {
```

We typically define these in singular, not plural.

```suggestion
    "RATING_BASE_BORDER_RADIUS": {
```